.github/workflows/release.yml

```
@@ -0,0 +1,73 @@
+name: Release
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  test:
+    uses: "ambanum/OpenTermsArchive/.github/workflows/test.yml@main"
+  release:
+    needs: [ test ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Git configuration
```

Let's use active verbs for all steps names 🙂 

```suggestion
      - name: Configure Git author
```